.licenserc.yaml

```diff
@@ -104,6 +104,8 @@ header:
     - 'dubbo-annotation-processor/src/main/java/org/apache/dubbo/annotation/permit/**'
     - 'dubbo-common/src/main/java/org/apache/dubbo/common/logger/helpers/FormattingTuple.java'
     - 'dubbo-common/src/main/java/org/apache/dubbo/common/logger/helpers/MessageFormatter.java'
+    - 'dubbo-maven-plugin/src/main/java/org/apache/dubbo/maven/plugin/protoc/DubboProtocCompilerMojo.java'
+    - 'dubbo-compiler/src/main/java/org/apache/dubbo/gen/utils/ProtoTypeMap.java'
```

Comment:
Please add the notification into `NOTICE` file to declare the copyright.

Reply:
ok,my pleasure